src/inference/tests/unit/query_model_test.cpp

```diff
@@ -520,3 +520,172 @@ TEST_F(GetSupportedNodesTest, AssignReadValueTest) {
         },
         {});
 }
+
+TEST_F(GetSupportedNodesTest, NoSupportedOpsTest) {
+    {
+        auto param = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, ov::PartialShape{1, 3, 2, 2});
+        param->set_friendly_name("input");
+        auto const_value = ov::op::v0::Constant::create(ov::element::f32, ov::Shape{1, 3, 2, 2}, {1});
+        const_value->set_friendly_name("const_val");
+        auto add = std::make_shared<ov::op::v1::Add>(param, const_value);
+        add->set_friendly_name("add");
+        auto res = std::make_shared<ov::op::v0::Result>(add);
+        res->set_friendly_name("res");
+        m_function = std::make_shared<ov::Model>(ov::ResultVector{res}, ov::ParameterVector{param});
+    }
+    Run(
+        [&](std::shared_ptr<ov::Model>& model) {
+            ov::pass::Manager m;
+            m.register_pass<ov::pass::InitNodeInfo>();
+            m.run_passes(model);
+        },
+        [&](const std::shared_ptr<ov::Node>& op) {
+            return false;
+        },
+        {},
+        0.9f);
+}
+
+TEST_F(GetSupportedNodesTest, NoConstOpTest) {
```

Comment:
This test is for a model which has no constant ops, because we get the size of model by constant ops, so just use this case to ensure it can not return unreasonable results.